### ISSUE TITLE
Can't see mapped network drives Windows 10 (current), Geany 1.37.1

### ISSUE THREAD

**Issue by BKeyport** [geany/geany#2794]
"Upgraded" from Geany 1.33, now Geany isn't able to access or use mapped network drives in the slightest.. 

The drives don't appear in Geany's open guide, even though everything else can access it. 

If you can give me some troubleshooting things to do, I can get you data to help solve it. 

Thanks!

**Comment by elextr** [comment]
Are you using native windows dialogs? Whichever try the opposite and see if the drives appear.

**Comment by BKeyport** [comment]
Using Native Windows Dialogs I get the drives showing like this: 

![image](https://user-images.githubusercontent.com/44744076/115638389-2cdffb00-a2c7-11eb-9674-5fcf4c49afe6.png)

**Comment by elextr** [comment]
Ok, so you have a workaround for now.

To help diagnose, what does the non-native dialog look like?

**Comment by BKeyport** [comment]
Oddly, when I switched back to non-native, it now shows correctly... So I'd consider it fixed..

**Comment by BKeyport** [comment]
I just switched to a newer computer, and downloaded the lastest software. Still doesn't show network drives upon initial load. 

Switching to native, closing the software and restarting, then switching back to your dialogs will fix it, however. 

Something's odd in the initial config?

**Comment by elextr** [comment]
@BKeyport please always post the top few lines of `help->debug messages` listing Geany version, Glib and GTK and OS.

**Comment by BKeyport** [comment]
18:59:21.475305: Geany INFO		: Using TCP port number 45937 for IPC
18:59:21.476305: Geany INFO		: Geany 1.38, English_United States.1252
18:59:21.476305: Geany INFO		: GTK 3.24.30, GLib 2.70.0
18:59:21.476305: Geany INFO		: OS: Windows 10 2009
18:59:21.476305: Geany INFO		: System data dir: C:\Program Files\Geany\data
18:59:21.476305: Geany INFO		: User config dir: C:\Users\brend\AppData\Roaming\geany

**Comment by elextr** [comment]
When you say "initial load" do you mean the first time the new version of Geany is opened ever, and it works from then on? Or the first time after the machine is rebooted? Or every time Geany is started?

**Comment by BKeyport** [comment]
Inital load meaning - first load after install, until I do the steps listed in my post you responded to, it works fine after that.

**Comment by elextr** [comment]
Did you open a file manager or anything else where the network drives are shown before trying Geany?  The GTK open dialog is from Linux where all network drives are mounted by the OS, so the dialog won't check for unmounted drives.  Maybe windows needs the file manager or open dialog to query them before it mounts them, dunno, not a Windows user.

**Comment by BKeyport** [comment]
Yes. In Windows, the "Mapped" drives are mounted upon start of the OS. There's no reason for it not to be there.

**Comment by elextr** [comment]
Somebody who can build on Windows could try replacing the GTKfilechooser with Nativefilechooser so it automatically uses the platform specific version.  Not sure if Geany uses anything thats not available on the native dialog, or if it works, or is useful for OSX or flatpack also?
